src/Symfony/Component/PropertyAccess/StringUtil.php

```diff
@@ -197,7 +202,7 @@ public static function singularify($plural)
         }
 
         // Convert teeth to tooth, feet to foot
-        if (false !== ($pos = strpos($plural, 'ee')) && strlen($plural) > 3) {
+        if (false !== ($pos = strpos($plural, 'ee')) && strlen($plural) > 3 ! in_array($plural, self::$allowedEes)) {
```

Comment:
@stof Yes, sorry, I'll be specific next time. 
Thanks, I'll implement the changes.

On second thought, maybe it would be easier to specify all "teeth, feet"-kind of words, than to allow all include "ee" + exclude others. What do you think?